projects/cashmere/src/lib/button/button.component.scss

```diff
@@ -39,6 +40,64 @@
     &.hc-link-inline {
         @include hc-button-link-inline();
     }
+
+    &.hc-button-blue {
+        @include hc-button-custom( $blue );
+    }
+    &.hc-button-green {
+        @include hc-button-custom( $green );
+    }
+    &.hc-button-purple {
+        @include hc-button-custom( $purple );
+    }
+    &.hc-button-red {
+        @include hc-button-custom( $red );
+    }
+    &.hc-button-orange {
+        @include hc-button-custom( $orange );
+    }
+    &.hc-button-ruby-red {
+        @include hc-button-custom( $ruby-red );
+    }
+    &.hc-button-deep-red {
+        @include hc-button-custom( $deep-red );
+    }
+    &.hc-button-red-orange {
+        @include hc-button-custom( $red-orange );
+    }
+    &.hc-button-magenta {
+        @include hc-button-custom( $magenta );
+    }
+    &.hc-button-pink {
+        @include hc-button-custom( $pink );
+    }
+    &.hc-button-light-pink {
+        @include hc-button-custom( $light-pink );
+    }
+    &.hc-button-azure {
+        @include hc-button-custom( $azure );
+    }
+    &.hc-button-teal {
+        @include hc-button-custom( $teal );
+    }
+    &.hc-button-dark-green {
+        @include hc-button-custom( $dark-green );
+    }
+    &.hc-button-brown {
+        @include hc-button-custom( $brown );
+    }
+    &.hc-button-purple-gray {
+        @include hc-button-custom( $purple-gray );
+    }
+    &.hc-button-yellow {
+        @include hc-button-custom( $yellow );
+    }
+    &.hc-button-yellow-orange {
+        @include hc-button-custom( $yellow-orange );
+    }
+    &.hc-button-tan {
```

Comment:
Given the programmatic capabilities of SCSS, there's probably a way to do all of these in a loop and save some space. This works though.